static/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -54,7 +55,18 @@ class ConfigureIntegration extends AsyncView<Props, State> {
   }
 
   componentDidMount() {
-    const {location} = this.props;
+    const {
```

Comment:
Euuughh, I litearlly undid https://github.com/getsentry/sentry/pull/31285 in https://github.com/getsentry/sentry/pull/31287 without noticing -.- rebasing is hard. 

No changes meant to be here, this shouldn't have been reverted in the second PR.